config.py

```diff
@@ -49,7 +49,13 @@ def read_configs(directory=CURRENT_DIRECTORY):
             configs = json.load(file)
 
     configs['device'] = os.getenv('MHP_CAMERA')
+    if not configs['device']:
+        print('WARNING: MHP_CAMERA has not been set in .env')
```

Comment:
Would this be handy in our situation? https://docs.python.org/3/library/warnings.html